File: iOS/Assassin/BaseLibrary/BaseLibrary/Classes/Modal/Account.swift
//
//  Account.swift
//  BaseLibrary
//
//  Created by 肖仲文 on 2021/1/8.
//

import HandyJSON

public class Account: HandyJSON {
    public var username: String?
    public var password: String?
    public var cnname: String?
    public var headUrl: String?

    required public init() {}
}

File: iOS/Assassin/assassin_flutter_module/lib/pages/me/models/account.dart
class Account {
  String username;
  String nickname;
  String headUrl;

  Account({this.username, this.nickname, this.headUrl});

  factory Account.fromJson(Map<String, dynamic> json) {
    return Account(
      username: json['username'],
      nickname: json['cnname'],
      headUrl: json['headUrl']
    );
  }
}